includes/admin/class-wc-admin-log-table-list.php

```diff
@@ -0,0 +1,356 @@
+<?php
+/**
+ * WooCommerce Log Table List
+ *
+ * @author   WooThemes
+ * @category Admin
+ * @package  WooCommerce/Admin
+ * @version  1.0.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! class_exists( 'WP_List_Table' ) ) {
+	require_once( ABSPATH . 'wp-admin/includes/class-wp-list-table.php' );
+}
+
+class WC_Admin_Log_Table_List extends WP_List_Table {
+
+	/**
+	 * Initialize the log table list.
+	 */
+	public function __construct() {
+		parent::__construct( array(
+			'singular' => __( 'log',  'woocommerce' ),
+			'plural'   => __( 'logs', 'woocommerce' ),
+			'ajax'     => false,
+		) );
+	}
+
+	/**
+	 * Get list views.
+	 *
+	 * @return array Views.
+	 */
+	public function get_views() {
```

Comment:
Source was originally a free text search, but now that we're using a dropdown filter for source, this should also be changed to match.